community/logging/src/main/java/org/neo4j/logging/RotatingFileOutputStreamSupplier.java

```diff
@@ -152,15 +154,22 @@ public void close() throws IOException
         synchronized (outRef)
         {
             closed.set( true );
-            outRef.get().close();
             for ( WeakReference<OutputStream> archivedStream : archivedStreams )
             {
                 OutputStream outputStream = archivedStream.get();
                 if ( outputStream != null )
                 {
-                    outputStream.close();
+                    try
+                    {
+                        outputStream.close();
+                    }
+                    catch ( Exception e )
+                    {
+                        // ignore
```

Comment:
I didn't think so. If there's any archived streams to close, it's only because they haven't been GC'd yet, and the GC would have swallowed and close exceptions.